### DISCUSSION TITLE
Add support of horizontal text alignement in description

### DISCUSSION THREAD

**Discussion by aacebedo** [structurizr/java#274] [Q&A]
### Description

It would be great if it was possible to choose the horizontal alignment of the text displayed in the description.
By default, it is centered, and it is not possible to choose left or right aligned.

### Priority

I have no budget and there's no rush, please add this feature for free

**Comment by simonbrowndotje** [comment]
I have no plans to add this - sorry!

**Reply by aacebedo** [reply]
Thanks for the answer.

If I look to add it through a PR, would it be accepted?
